docs/contributing/contributing_pl.md

```diff
@@ -66,7 +66,7 @@ done
 # Usuń package-lock w podkatalogach
 find . -type f -name "package-lock.json" -delete
 
-```
+````
```

Comment:
Same here.

Reply:
Thanks! Didn't notice that after merge. I used merge in WebStorm. It seems, that it was modified by git hooks, which formatted this braces due to lack of `` ``` `` for `` ```bash ``. I think it tried to convert:

````
```bash
```shell
```
````

to

`````
````bash
```shell
````
`````

I added missed `` ``` `` and changed `` ```` `` back to `` ``` ``.

Reply:
It looks like someone already pushed this typo in main branch - you can see `` ```` `` in this [file](https://github.com/usebruno/bruno/blob/main/docs/contributing/contributing_pl.md?plain=1). In this PR I squished some commits, so I did not include this fix because it is not related to the changes I made